src/EditorFeatures/Test2/IntelliSense/CSharpCompletionCommandHandlerTests.vb

```diff
@@ -390,7 +390,8 @@ class Variable
             End Using
         End Function
 
-        <MemberData(NameOf(AllCompletionImplementations))>
+        <InlineData(CompletionImplementation.Legacy)>
+        <InlineData(CompletionImplementation.Modern, Skip:="https://github.com/dotnet/roslyn/issues/27454")>
```

Comment:
why do we have skipped tests?

Reply:
We have an open issue ([https://github.com/dotnet/roslyn/issues/27454](https://github.com/dotnet/roslyn/issues/27454)) to be fixed before setting the feature flag ON

---
In reply to: [222522942](https://github.com/dotnet/roslyn/pull/29016#discussion_r222522942) [](ancestors = 222522942)